README.md

```diff
@@ -114,21 +118,26 @@ If you want to see help, run `budoux -h`.
 
 ```shellsession
 $ budoux -h
-usage: budoux [-h] [-H] [-m JSON] [-d STR] [-t THRES] [-V] [TXT]
+usage: budoux [-h] [-H] [-m JSON | -l LANG] [-d STR] [-t THRES] [-V] [TXT]
 
 BudouX is the successor to Budou,
 the machine learning powered line break organizer tool.
 
 positional arguments:
   TXT                      text (default: None)
 
-optional arguments:
+options:
   -h, --help               show this help message and exit
   -H, --html               HTML mode (default: False)
-  -m JSON, --model JSON    custom model file path (default: /path/to/models/ja-knbc.json)
+  -m JSON, --model JSON    custom model file path (default: /home/eggplants/prog/budoux/budoux/models/ja-knbc.json)
```

Comment:
Probably we may want to generalize the default model path.

Reply:
1c03f6e